src/expr.rs

```diff
@@ -138,7 +138,19 @@ fn format_expr_inner(
         ast::ExprKind::Call(ref callee, ref args) => {
             let inner_span = mk_sp(callee.span.hi(), expr.span.hi());
             let callee_str = callee.rewrite(context, shape)?;
-            rewrite_call(context, &callee_str, args, inner_span, shape)
+            let force_list_tactic = if context.config.was_set().fn_call_layout() {
+                Some(context.config.fn_call_layout().to_list_tactic(args.len()))
+            } else {
+                None
+            };
```

Comment:
Would you mind elaborating on your thinking with this behavior for me? Something that would be worrisome is if there's the potential for someone to run `rustfmt foo.rs` with no config file/no config overrides and get one result, and then run `rustfmt foo.rs --config fn_call_layout=Tall` and get a different result.

Are there scenarios where you could imagine that happening due to the keying on `was_set`?

Reply:
To be honest, I'm not sure why I wrote it that way. I don't think there would be any issues with removing the conditional and always setting `force_list_tactic=Some(context.config.fn_call_layout().to_list_tactic(args.len()))`